### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/chart/mermaid.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#859] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/chart/mermaid.html

让你的 VuePress 站点中的 Markdown 文件支持 mermaid。 配置 在你的项目中安装 mermaid: 之后启用它: 语法 除了使用 mermaid 代码块，你也可以直接使用以下代码块： class: classDiagram c4c: C4Context er: erDiagram gantt: gantt git-graph: ...

https://plugin-md-enhance.vuejs.press/zh/guide/chart/mermaid.html

**Comment by nfzwyio** [comment]
nice!
